devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -437,13 +437,20 @@ bool HumanStateProvider::open(yarp::os::Searchable& config)
     }
 
     std::vector<double> calibrationJointConfiguration;
-    if (!(config.check("calibrationJointConfiguration") && config.find("calibrationJointConfiguration").isList())) {
+    if (!config.check("calibrationJointConfiguration"))
+    {
         yInfo() << LogPrefix << "calibrationJointConfiguration option not found or not valid, using zero configuration instead.";
```

Comment:
```suggestion
        yInfo() << LogPrefix << "calibrationJointConfiguration option not found, using zero configuration instead.";
```

We moved the "not valid case" in the other branch of if.